### ISSUE TITLE
Failed to initialize the add-in "PyRevitLoader"...

### ISSUE THREAD

**Issue by Tyree** [eirannejad/pyRevit#1670]
Same as this thread: https://github.com/eirannejad/pyRevit/issues/1381

![Screenshot 2022-11-28 073727](https://user-images.githubusercontent.com/5395363/204279757-222cd6d4-a095-4a4c-892f-09cf65aa67eb.png)

I can't see the entire path of the missing dll, so I can't confirm if it's there or not. I have uninstalled and reinstalled pyRevit since I don't know why it's installing to the Administrator user when I'm logged in as another admin user.

**Comment by jmcouffin** [comment]
and same as Ehsan mentioned Please run pyrevit env on command line and share the results here

**Comment by Tyree** [comment]
I THINK this may be because I used a new Windows custom installer with Revit and pyRevit preinstalled. So, it was originally installed under the Administrator user on the tech machine and now that it's on a live machine pyRevit is hung thinking it needs to use the original Administrator location that Revit was installed under. I thought uninstalling and reinstalling pyRevit would fix that, but I guess not.
I installed Revit 2021 after the Windows install and the same pyRevit install loads fine there. It's just Revit 2022 that was installed with Windows that has the issue.

So, I'm uninstalling and reinstalling Revit 2022 and pyRevit to see if that solves it. I'll report back.
